Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -533,7 +535,7 @@ public void indexConcepts(String userName)
                 String searchString = fieldMap.get(search.fieldName());
                 if (searchString != null) {
                     searchString = searchString.toLowerCase();
-                    buildQuery(occur, perFieldAnalyzerWrapper, qBuild, builder, luceneFieldAnnotation, searchString);
+                    buildQuery(occur, perFieldAnalyzerWrapper, qBuild, builder, luceneFieldAnnotation, searchString, isSearchOnDescription);
```

Comment:
isn't the only thing you need to do, test if `isSearchOnDescription` is true and if search is the description field, then exclude or not exclude it here?

Reply:
something like 
```
if (search.fieldName != SearchFieldNames.DESCRIPTION || isSearchOnDescription ) { // or however you get the filed name
 buildQuery(...)
}
```